venmail.io.venmail_email.json

```diff
@@ -0,0 +1,59 @@
+{
+    "providerId": "venmail.io",
+    "providerName": "Venmail",
+    "serviceId": "venmail_email",
+    "serviceName": "Venmail",
+    "logoUrl": "https://m.venmail.io/logo.png",
+    "description": "This is a template to enable mail hosting with venmail",
+    "variableDescription": "",
+    "syncBlock": false,
+    "hostRequired": false,
+    "version": 1,
+    "syncPubKeyDomain": "venmail.io",
+    "records": [
+    {
+        "groupId": "venmail-dkim",
+        "type": "TXT",
+        "host": "%dkim_name%._domainkey",
+        "data": "%dkim_data%",
+        "ttl": 600
+    },
+    {
+        "groupId": "website-redirection",
+        "type": "CNAME",
+        "host": "%website_host%",
+        "pointsTo": "%website_points%",
+        "ttl": 600
+    },
+    {
+        "groupId": "venmail-mx",
+        "type": "MX",
+        "host": "@",
+        "pointsTo": "%mx_points%",
+        "priority": 10,
+        "ttl": 600
+    },
+    {
+        "groupId": "venmail-mx2",
+        "type": "MX",
+        "host": "@",
+        "pointsTo": "%mx2_points%",
+        "priority": 20,
+        "ttl": 600
+    },
+    {
+        "groupId": "venmail-spf",
+        "type": "TXT",
+        "host": "@",
+        "data": "%spf_data%",
```

Comment:
Please use SPFM instead of pure TXT record.

Reply:
Done

Reply:
@pawel-kow for clarification, some additional info shows traditional spf is deprecated? Is this different from SPFM implementation? https://mxtoolbox.com/problem/spf/spf-record-deprecated

Reply:
Please read the spec what SPFM is.
SPF deprecation is about dedicated SPF RR which was replaced by TXT record.

I would assume you want to remove group venmail-spf after reading the spec.

Reply:
Seen quite interesting. We were considering scanning the existing records and regenerating but truly the DNS provider is in a better position for that. Thanks!